kaldb/src/test/java/com/slack/kaldb/metadata/core/KaldbMetadataStoreTest.java

```diff
@@ -33,6 +35,12 @@
 
 @RunWith(Enclosed.class)
 public class KaldbMetadataStoreTest {
+
+  private static final String SNAPSHOTS_PATH = "/snapshots";
```

Comment:
This should probably reuse the config value of `SNAPSHOT_METADATA_STORE_ZK_PATH` instead of redeclaring it here with the same value.

Reply:
This code was written before those variables are defined. Agree on reusing. Updated the code.